docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  riak:
+    image: basho/riak-kv:ubuntu-2.2.3
```

Comment:
Maybe use our [riak image](https://github.com/kivra/kivra-in-a-box/blob/main/docker-compose.base.yml#L172) instead?

Reply:
Sure. I got this reference from Heimdall (only added the version, really), so I should update that in the future, as well...

Reply:
Wait, this is a public repository. Is our image public too?

Reply:
Ahh, you are right, I didn't think about that. No, the image is not public I think, so ignore this comment